src/index.css

```diff
@@ -55,22 +98,78 @@ input {
 }
 
 .game {
+  background-color: #f6f7fb;
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
   display: flex;
   justify-content: center;
+  align-items: center;
 }
 
-.game-info {
-  display: flex;
+.game__info {
+  background-color: #ffffff;
+  border-radius: 6px;
+  box-shadow: rgba(0, 0, 0, 0.1) 0px 3px 10px;
+  max-width: 500px;
+  overflow: hidden;
   justify-content: center;
   align-items: center;
   flex-direction: column;
+  width: 100%;
 }
 
-.game-player {
-  margin-bottom: 10px;
+.game__title {
+  justify-content: center;
+  display: flex;
+  flex-wrap: wrap;
+  color: #ffffff;
+  background-color: #43d1af;
+  margin: 0;
+  box-shadow: rgba(0, 0, 0, 0.1) 0px 1px 5px;
+  z-index: 1;
+  padding: 16px;
+  text-align: center;
+  font-weight: 600;
 }
 
-.game-status {
+.game__content {
   font-size: 20px;
   margin-bottom: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 16px;
+}
+
+.game__player {
+  width: 100%;
```

Comment:
I think a lot of these `width: 100%` can be removed, most of them are `div` elements which would inherently have that.

Reply:
Take a look at all the places you define that and see if they can be removed without affecting your layout?

Reply:
I looked and the only one that will affect the layout would be the Game. I did not remove that one